hack/test-end-to-end-docker.sh

```diff
@@ -30,7 +30,17 @@ if [[ -z "${BASETMPDIR-}" ]]; then
   sudo rm -rf "${BASETMPDIR}"
   mkdir -p "${BASETMPDIR}"
 fi
-VOLUME_DIR="${BASETMPDIR}/volumes"
+
+# when selinux is enforcing, the volume dir selinux label needs to be
+# svirt_sandbox_file_t
+#
+# TODO: fix the selinux policy to either allow openshift_var_lib_dir_t
+# or to default the volume dir to svirt_sandbox_file_t.
+sudo mkdir -p /var/lib/openshift/openshift.local.volumes
```

Comment:
This is the way this script was originally set up. I tried switching to use VOLUME_DIR under /tmp/openshift-e2e. It took me a while to figure out why the mount calls were failing. They use `nsenter_mount.go`, which runs `nsenter` using the mount namespace from the node's pid 1. I think it wasn't able to see the dir, but I can dig further if needed.